src/test/java/org/dependencytrack/metrics/MetricsTest.java

```diff
@@ -18,10 +18,23 @@
  */
 package org.dependencytrack.metrics;
 
+import org.dependencytrack.persistence.QueryManager;
+
 import org.junit.Assert;
 import org.junit.Test;
 
-public class MetricsTest {
+import alpine.model.IConfigProperty;
+
+import static org.dependencytrack.model.ConfigPropertyConstants.CUSTOM_RISK_SCORE_CRITICAL;
+import static org.dependencytrack.model.ConfigPropertyConstants.CUSTOM_RISK_SCORE_HIGH;
+import static org.dependencytrack.model.ConfigPropertyConstants.CUSTOM_RISK_SCORE_MEDIUM;
+import static org.dependencytrack.model.ConfigPropertyConstants.CUSTOM_RISK_SCORE_LOW;
+import static org.dependencytrack.model.ConfigPropertyConstants.CUSTOM_RISK_SCORE_UNASSIGNED;
+
+import org.dependencytrack.PersistenceCapableTest;
+
+public class MetricsTest extends PersistenceCapableTest{
+    protected QueryManager qm;
```

Comment:
Removing this should resolve the current error in CI. `qm` should be inherited from `PersistenceCapableTest` and be accessible to you in your test method.

Reply:
removed, thanks